src/anomalib/models/uflow/README.md

```diff
@@ -0,0 +1,109 @@
+# U-Flow: A U-shaped Normalizing Flow for Anomaly Detection with Unsupervised Threshold.
+
+[//]: # (This is the implementation of the [U-Flow]&#40;https://arxiv.org/abs/2211.12353&#41; paper, based on the [original code]&#40;https://www.github.com/mtailanian/uflow&#41;)
+
+This is the implementation of the [U-Flow](https://www.researchsquare.com/article/rs-3367286/latest) paper, based on the [original code](https://www.github.com/mtailanian/uflow)
+
+![U-Flow Architecture](https://raw.githubusercontent.com/openvinotoolkit/anomalib/main/docs/source/images/uflow/diagram.png "U-Flow Architecture")
+
+## Abstract   
+
+_In this work we propose a one-class self-supervised method for anomaly segmentation in images, that benefits both from a modern machine learning approach and a more classic statistical detection theory. The method consists of three phases. First, features are extracted using a multi-scale image Transformer architecture. Then, these features are fed into a U-shaped Normalizing Flow that lays the theoretical foundations for the last phase, which computes a pixel-level anomaly map and performs a segmentation based on the a contrario framework. This multiple-hypothesis testing strategy permits the derivation of robust automatic detection thresholds, which are crucial in real-world applications where an operational point is needed. The segmentation results are evaluated using the Intersection over Union (IoU) metric, and for assessing the generated anomaly maps we report the area under the Receiver Operating Characteristic curve (AUROC), and the area under the per-region-overlap curve (AUPRO). Extensive experimentation in various datasets shows that the proposed approach produces state-of-the-art results for all metrics and all datasets, ranking first in most MvTec-AD categories, with a mean pixel-level AUROC of 98.74%._
+
+![Teaser image](https://raw.githubusercontent.com/openvinotoolkit/anomalib/main/docs/source/images/uflow/teaser.png)
+
+## Localization results
+### Pixel AUROC over MVTec-AD Dataset
+
+![Pixel-AUROC results](https://raw.githubusercontent.com/openvinotoolkit/anomalib/main/docs/source/images/uflow/results/pixel-auroc.png "Pixel-AUROC results")
+
+### Pixel AUPRO over MVTec-AD Dataset
+
+![Pixel-AUPRO results](https://raw.githubusercontent.com/openvinotoolkit/anomalib/main/docs/source/images/uflow/results/pixel-aupro.png "Pixel-AUPRO results")
```

Comment:
Same as above, I can only see `diagram.png` and `teaser.jpg`

Reply:
Same!